sdk/schemaregistry/azure-schemaregistry/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## 1.0.1 (Unreleased)
 
+This version and all future versions will require Python 3.7+. Python 2.7 and 3.6 are no longer supported.
```

Comment:
Is this the only change for the release? If so, there's no need to release.

Reply:
sounds good, just will update the version and leave this note for now. Not planning on releasing